Open new Discord server (#48)
rails/website: _posts/2022-06-13-rails-discord-server-is-now-open-to-the-public.markdown

---
layout: post
title: "Rails Discord server is now open to the public"
categories: news
author: rafaelfranca
published: true
date: 2022-06-13
---

> In the past few weeks, the Rails team has been working in a few changes to lower the barrier for new contributors to the framework.

Reviewer: ```suggestion
In the past few weeks, the Rails team has been working on a few changes to lower the barrier for new contributors to the framework.
```

> In the past few weeks, the Rails team has been working in a few changes to lower the barrier for new contributors to the framework.

Reviewer 2: ```suggestion
In the past few weeks, the Rails team has been working on a few changes to lower the barrier for new contributors to the framework.
```

We started by [disabling the automatic closing of stale PRs](https://github.com/rails/rails/commit/acf48169943011834c4c885b956e29050548fa98). This means that contributors can submit their PRs without

> the fear of a bot closing their worth contributions before a member of the Rails Team has a chance to review them.

Reviewer: ```suggestion
the fear of a bot closing their worthy contributions before a member of the Rails Team has a chance to review them.
```

> the fear of a bot closing their worth contributions before a member of the Rails Team has a chance to review them.

Reviewer 2: ```suggestion
the fear of a bot closing their worthy contributions before a member of the Rails Team has a chance to review them.
```

> the fear of a bot closing their worthy contributions before a member of the Rails Team has a chance to review them.

Reviewer 3: Most usages of Rails team is without a capitalized team, so changing to be consistent:
```suggestion
the fear of a bot closing their worthy contributions before a member of the Rails team has a chance to review them.
```

> To make easier for contributors to recognize members of the Rails Team, we made public the [list

Reviewer 2: ```suggestion
To make it easier for contributors to recognize members of the Rails Team, we made public the [list
```

> To make easier for contributors to recognize members of the Rails Team, we made public the [list

Reviewer: ```suggestion
To make easier it for contributors to recognize members of the Rails Team, we made public the [list
```

> To make it easier for contributors to recognize members of the Rails Team, we made public the [list

Reviewer 3: I think this sentence is trying to say that listing the issues team members made it easier to recognize the "whole" team, which can be made clearer:
```suggestion
To make it easier for contributors to recognize all the members of the Rails team, we made public the [list
```

of current members of the Issues team](https://rubyonrails.org/community). The Issues team assists with
issues triage, pull request reviews and documentation improvements, so they are often the first

> interaction of users with the Rails team.

Reviewer 2: ```suggestion
interaction users have with the Rails team.
```

Today, we are opening a Discord server to allow contributors to help each other and lower the overhead

> to communicate with new contributors.

Reviewer 3: ```suggestion
of communicating with new contributors.
```

> This new server is open to the public using the following [this invite link](https://discord.gg/8N68BCw49),

Reviewer: ```suggestion
This new server is open to the public using [this invite link](https://discord.gg/8N68BCw49),
```

> This new server is open to the public using the following [this invite link](https://discord.gg/8N68BCw49),

Reviewer 2: ```suggestion
This new server is open to the public using [this invite link](https://discord.gg/8N68BCw49),
```

> and is a great place to get help with the framework and how to contribute to it and will be complement

Reviewer 2: ```suggestion
and is a great place to get help with the framework and how to contribute to it and will complement
```

> and is a great place to get help with the framework and how to contribute to it and will be complement

Reviewer: ```suggestion
and is a great place to get help with the framework and learn how to contribute to it. This will also complement
```

the [GitHub issue tracker](https://github.com/rails/rails/issues) and our
[Discourse forum](https://discuss.rubyonrails.org/). It will follow the same

> [Code Of Conduct](https://rubyonrails.org/conduct) of the rest of the Rails official spaces.

Reviewer 2: ```suggestion
[Code Of Conduct](https://rubyonrails.org/conduct) as the rest of the Rails official spaces.
```

---
layout: post
title: "Rails Discord server is now open to the public"
categories: news
author: rafaelfranca
published: true
date: 2022-06-13
---

In the past few weeks, the Rails team has been working on a few changes to lower the barrier for new contributors to the framework.

> We started by [disabling the automatic closing of stale PRs](https://github.com/rails/rails/commit/acf48169943011834c4c885b956e29050548fa98). This means that contributors can submit their PRs without
> the fear of a bot closing their worthy contributions before a member of the Rails team has a chance to review them.

Reviewer 4: I'd like to propose a more positive wording here, without mentioning negative emotions. Also, you could have a PR reviewed by a team member, but then stalled too, the bot did not prevent that.

Something like:

```suggestion
We started by [disabling the automatic closing of stale PRs](https://github.com/rails/rails/commit/acf48169943011834c4c885b956e29050548fa98). While initially this automation helped the project keep the issues tracker under control, nowadays things are better and we prefer to go back to full human interaction.
```
